### DISCUSSION TITLE
LiveConnect Client Module

### DISCUSSION THREAD

**Discussion by verdeanSICKAG** [SICKAppSpaceCodingStarterKit/.github#3] [Ideas]
Hey guys,

I will create a CSK module to bring data into the cloud using SICK LiveConnect and the SICK AssetHub. Can you provide a repository where I can add my stuff to share it our customer?

**Comment by golluroSICKAG** [comment]
Hi!
Is it ok to name it "CSK_Module_LiveConnectClient"? Or any other suggestion?
Kind regards!

**Reply by verdeanSICKAG** [reply]
Please name it "CSK_Module_LiveConnect".

Thanks

**Comment by golluroSICKAG** [comment]
Just created the repo here:
https://github.com/SICKAppSpaceCodingStarterKit/CSK_Module_LiveConnect
Happy coding!

**Comment by golluroSICKAG** [comment]
Thank you very much for your contribution to  

[CSK_Module_LiveConnect](https://github.com/SICKAppSpaceCodingStarterKit/CSK_Module_LiveConnect)  
and  
[CSK_Template_LiveConnectDemo](https://github.com/SICKAppSpaceCodingStarterKit/CSK_Template_LiveConnectDemo)  

I will close this discussion now.
